tests/components/husqvarna_automower_ble/conftest.py

```diff
@@ -0,0 +1,114 @@
+"""Common fixtures for the Husqvarna Automower Bluetooth tests."""
+
+from collections.abc import Awaitable, Callable, Generator
+from unittest.mock import AsyncMock, patch
+
+from freezegun.api import FrozenDateTimeFactory
+import pytest
+
+from homeassistant.components.husqvarna_automower_ble.const import DOMAIN
+from homeassistant.components.husqvarna_automower_ble.coordinator import SCAN_INTERVAL
+from homeassistant.const import CONF_ADDRESS, CONF_CLIENT_ID
+from homeassistant.core import HomeAssistant
+
+from . import AUTOMOWER_SERVICE_INFO
+
+from tests.common import MockConfigEntry, async_fire_time_changed
+
+
+@pytest.fixture
+def mock_entry():
+    """Create hass config fixture."""
+    return MockConfigEntry(
```

Comment:
I think we have an unique id in real life, so let's make this mock config entry reflect real life

Reply:
Unless I'm misunderstanding I don't think we do